registry_pvc.yaml

```diff
@@ -5,8 +5,7 @@ metadata:
   namespace: openshift-image-registry
 spec:
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
       storage: 20Gi
-  storageClassName: crc-csi-hostpath-provisioner
```

Comment:
The commit log could detail that this also removes the configuration of the hostpath-provisioner and explain why by removing `storageClassName` the image registry starts using topolvm.